arcade/types.py

```diff
@@ -49,6 +52,22 @@
 
 RGBA255OrNormalized = Union[RGBA255, RGBANormalized]
 
+# equivalent to the following filters (under arcade.gl.enums):
+# NEAREST, LINEAR, NEAREST_MIPMAP_NEAREST, LINEAR_MIPMAP_NEAREST,
+# NEAREST_MIPMAP_LINEAR, LINEAR_MIPMAP_LINEAR
+OpenGlFilters: TypeAlias = Union[Literal[9728], Literal[9729], Literal[9984],
```

Comment:
```suggestion
OpenGlFilter: TypeAlias = Union[Literal[9728], Literal[9729], Literal[9984],
```
Since it's a single object, should this be singular? `ArcadeContext` seems to confirm it as such given its `blend_func` instance variable is singular.